README.md

```diff
@@ -73,39 +39,48 @@ You can try Cadl on the web without installing anything.
 
    It is recommended to have npm 7+. To update npm run `npm install -g npm`
 
-2. Create a folder for your new Cadl project
+2. Install Cadl compiler and libraries:
 
-3. **Via init command:** Run `npx -p @cadl-lang/compiler cadl init` > Select openapi3 library template.
-
-4. **Alternatively manually:** In a command prompt, run the following commands:
-
-   ```
-   cd path\to\cadl\project
+```bash
    npm init -y
    npm install -g @cadl-lang/compiler
-   npm install @cadl-lang/rest @cadl-lang/openapi3
-   ```
+```
 
-   This will create a `package.json` file for your Cadl project and add the necessary Cadl dependencies to it.
+If you do not wish to install the compiler globally with `-g` flag, you will need to install it locally once in every Cadl project folder. You would also need to prefix every Cadl run command with `npx`. See [npx documentation](https://docs.npmjs.com/cli/v7/commands/npx)
 
-5. Install the Cadl extension for your editor of choice:
+```bash
+    npx cadl init
+    npx cadl compile
+```
+
+3. Install the Cadl extension for your editor of choice:
 
    - [Instructions for Visual Studio](#installing-visual-studio-extension)
    - [Instructions for Visual Studio Code](#installing-vs-code-extension)
 
-6. Open the folder in your editor and create a new file `main.cadl`
+### Creating Cadl project
+
+1. Create a folder for your new Cadl project and install libraries in local folder:
 
-7. Follow our [documentation](https://microsoft.github.io/cadl) to get started writing Cadl!
+```bash
+   npm install @cadl-lang/rest @cadl-lang/openapi3
+```
 
-8. Once you're ready to compile your Cadl to Swagger, save the file and type this at the command prompt in your project folder:
+2. Initialize a Cadl project. Run `cadl init` > Select openapi3 library template. This will create a `package.json` file for your Cadl project and add the necessary Cadl dependencies to it.
 
-   ```
-   npx cadl compile . --emit @cadl-lang/openapi3
+3. Open the folder in your editor and create a new file `main.cadl`
+
+4. Follow our [documentation](https://microsoft.github.io/cadl) to get started writing Cadl!
+
+5. Once you're ready to compile your Cadl to Swagger, save the file and type this at the command prompt in your project folder:
+
+   ```bash
+   cadl compile . --emit @cadl-lang/openapi3
```

Comment:
cadl init will generate a cadl-project.yaml with the openapi3 emitter already configured. So I don't think you need to specify it explicitly here.